eve/single_node/group_vars/k8s-cluster/single_node.yml

```diff
@@ -0,0 +1,24 @@
+
+metal_k8s_lvm:
+  vgs:
+    kubevg:
+      drives: ['/dev/loop0']
+
+metal_k8s_storage_class:
+  storage_classes:
+    local-lvm:
+      is_default: true
+  lvm_conf:
+    default_fstype: 'ext4'
```

Comment:
Must all of these be set when we could rely on defaults?

Reply:
but defaults are replaced not merged. We should convert this in a less tree-ish way, more flat. This allow to override just one key.